apps/webapp/web/static/js/docs/index.js

```diff
@@ -0,0 +1,52 @@
+// file: docs/index.js
+import React from 'react';
+import mdlUpgrade from 'lib/mdlUpgrade';
+import material from 'material-design-lite/material.css';
+import styles from 'docs/styles.css';
+import TableOfContents from 'docs/TableOfContents';
+import DocPage from 'docs/DocPage';
+import fetch from 'isomorphic-fetch';
+
+class Index extends React.Component {
+
+  constructor(props) {
+    super();
+    this.state = {path: 'index', docs: []};
+  }
+
+  componentDidMount() {
+    // TODO: change window.location.origin to a URL variable rendered from the server.
+    fetch(window.location.origin + '/docs/index')
+      .then(response => {
+        if (response.status >= 400) {
+          throw new Error("Bad response from server");
+        }
+        return response.json();
+      })
+      .then(list => {
+        this.setState({ docs: list });
+      });
+  }
+
+  changeDocument (e) {
+    e.preventDefault();
+    this.setState({
+      path: e.target.getAttribute('data-url')
```

Comment:
@rsjavier14  Actually, you don't need to get `data-url` from `e.target`. You can make `changeDocument(docIndex)`. The implementation receiving this callback would pass the corresponding `docIndex`.